include/cvc5/cvc5_parser.h

```diff
@@ -325,6 +330,10 @@ class CVC5_EXPORT InputParser
   std::stringstream d_istringStream;
   /** Are we initialized to use the above string stream? */
   bool d_usingIStringStream;
+  /** The last language argument passed to setIncrementalStringInput */
```

Comment:
```suggestion
  /** The last language argument passed to setIncrementalStringInput(). */
```
@ajreynol you have to use `()` for function references for Doxygen to pick this up properly and link to the function.